frontend/app/templates/components/badge-design.hbs

```diff
@@ -0,0 +1,35 @@
+{{!--
+<div class="ui grid">
+  <div class="six wide column">
+
+  </div>
+  <div class="six wide column">
+    <h3>Features</h3>
+  </div>
+</div> --}}
+<div class="wrapper">
+          <br>
+        <br>
+        <p class="heading-alpha">Features</p>
+        <br>
+  <div class="ui stackable two column grid container center aligned">
+    <div class="column">
+      <div class="carousel-body">
+        <img class="ui large image" src="https://raw.githubusercontent.com/rajvaibhavdubey/rajvaibhavdubey.github.io/master/badge_design_red.png">
+      </div>
+    </div>
+    <div class="column">
+      <br>
+      <br>
+      <br>
+      <div class="border-left"></div>
+      <div class="ui left aligned bulleted list">
+        <h3 class="item">Many badge backgrounds to choose from</h3>
+        <h3 class="item">Different types of available badge sizes</h3>
+        <h3 class="item">Custom font colours</h3>
+        <h3 class="item">Custom background sources</h3>
+        <h3 class="item">Different paper sizes</h3>
+      </div>
+    </div>
+  </div>
+  </div>
```

Comment:
```suggestion
  </div>
  
```
LGTM 👍, it makes sense to have this feature. It will be good for new users to see what badeyay offers you, without even making an account for the same.